plottr/data/qcodes_dataset.py

```diff
@@ -252,3 +255,5 @@ def process(self, **kw):
                 data.add_meta('qcodes_runTS', ds.run_timestamp())
                 self.nLoadedRecords = ds.number_of_results
                 return dict(dataOut=data)
+        else:
+            return None
```

Comment:
hmm, don't we also need a `return None` for the `else` clause of the `if ds.number_of_results > self.nLoadedRecords:`?

Reply:
Yes you are right I missed that one